angular.json

```diff
@@ -46,12 +51,12 @@
               "outputHashing": "all",
               "sourceMap": false,
               "namedChunks": false,
-              "aot": true,
               "extractLicenses": true,
               "vendorChunk": false,
               "buildOptimizer": true
             }
-          }
+          },
+          "defaultConfiguration": ""
```

Comment:
Might also be a good idea to run optional "production-by-default" migration:
`ng update @angular/cli@12 --migrate-only production-by-default`

Reply:
Thank you for the notice. Was so used to not having the flag anymore, I didn't notice it was here.
Fixed